Owncloud iOs Client/Tabs/SettingTab/SettingsViewController.m

```diff
@@ -1762,260 +1769,21 @@ -(void)setPropertiesTouchIDToState:(BOOL)isTocuhIDActive{
     }
 }
 
-
-#pragma mark - Instant Upload - Location Support
-
-- (void)switchInstantUploadTo:(BOOL)value {
-     [self.switchInstantUpload setOn:value animated:NO];
-}
-
--(void)setPropertiesInstantUploadToState:(BOOL)stateInstantUpload{
-    
-    AppDelegate *app = (AppDelegate *)[[UIApplication sharedApplication] delegate];
-    if (app.activeUser.username == nil) {
-        app.activeUser = [ManageUsersDB getActiveUser];
-    }
-    
-    if (stateInstantUpload) {
-        [self switchInstantUploadTo:YES];
-        app.activeUser.instantUpload = YES;
-        [ManageAppSettingsDB updateInstantUploadTo:YES];
-    } else {
-        [self switchInstantUploadTo:NO];
-        app.activeUser.instantUpload = NO;
-        [ManageAppSettingsDB updateInstantUploadTo:NO];
-        [[ManageLocation sharedSingleton] stopSignificantChangeUpdates];
-    }
-}
-
--(void)setDateInstantUpload{
```

Comment:
What @javiergonzper meant here is once an user active the instant uploads, dateInstantUpload (now called timestampInstantUpload) should be updated in db and active userDTO with the current date, in order to instant upload only all the new photos. So, when you check in attempUpload the date you already have a previous date with which to compare.